templates/rustdoc/topbar.html

```diff
@@ -226,7 +226,27 @@
                 </li>
             {%- endfor -%}
         </ul>
-    </li>
+    </li>{#
+    Display the features available in current build
+    #}
+    {%- if metadata.features -%}
+        <li class="pure-menu-item pure-menu-has-children">
+            <a href="#" class="pure-menu-link" aria-label="Features">
+                {{ "flag" | fas }}
+                <span class="title">Feature flags</span>
+            </a>
+
+            {# Build the dropdown list showing available features #}
+            <ul class="pure-menu-children">
+                {%- for feature in metadata.features -%}
+                {%- set target_url = "/crate/" ~ metadata.name ~ "/" ~ metadata.version ~ "/features#" ~ feature.name -%}
+                    <li class="pure-menu-item">
+                        <a href="{{ target_url | safe }}" class="pure-menu-link">{{ feature.name }}</a>
+                    </li>
+                {%- endfor -%}
+            </ul>
```

Comment:
This gets a little overwhelming with a lot of features. Can you have just a single link to `/crate/.../features` instead, the way you have it on /crate?

![image](https://user-images.githubusercontent.com/23638587/98629336-a28e7680-22e6-11eb-91cf-910f8640a9cb.png)

Reply:
Done (Hopefully I understood it well)